### PR TITLE
Update PDF2IMG documentation for 4.7.0 release

### PR DESCRIPTION
**Pull request by datalogics-saharay**

### CODE REVIEW

File: PDF2IMG/index.html [hunk 1]
```diff
@@ -19,6 +19,8 @@
   <h1>Datalogics PDF2IMG Documentation</h1>
  <h2>PDF2IMG</h2>
  <p>A command-line utility and API that converts PDF files to a variety of image formats, including PNG, JPG, TIFF, BMP, and more. The product is built upon the Adobe PDF Library and uses Adobe technology for unrivaled color management during the PDF conversion process. While PDF2IMG can be run with as few as two command line arguments, many other optional command line arguments are provided so that you can create custom output. PDF2IMG is available on both Windows and Linux platforms, 32 and 64 bit.</p>
+ <br>
+ <p>PDF2IMG's C# interface is now available as a x64 Windows NuGet package.</p>
```
Comment (datalogics-robb) [inline PDF2IMG/index.html:23]: should this be "as an x64"?

Comment (datalogics-saharay) [reply PDF2IMG/index.html:23]: No, I think "as a x64" is correct because the word following  "a" isn't a vowel.

Comment (datalogics-robb) [reply PDF2IMG/index.html:23]: but the syllable is a vowel sound - "Ex"

**Comment by datalogics-ganthony** [comment]
Looks good

**Pull request merged**